go_router/test/go_router_test.dart

```diff
@@ -9,8 +9,19 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:go_router/go_router.dart';
 import 'package:go_router/src/go_route_match.dart';
 import 'package:go_router/src/go_router_refresh_stream.dart';
+import 'package:logging/logging.dart';
+
+const enableLogs = false;
+final log = Logger('GoRouter tests');
 
 void main() {
+  if (enableLogs) {
+    Logger.root.onRecord.listen((e) {
+      // TODO(johnpryan): why doesn't log() from dart:developer print anything?
```

Comment:
can we answer this question, do the right thing in the implementation of this method and remove the TODO statement? @johnpryan @pq

Reply:
Not sure what's going on here...  Maybe @devoncarew has thoughts?

Reply:
It might be WAI... Does it make sense for `dart:developer`'s `log()` to print to stdout when  the `flutter test` command is used?

Reply:
It looks like I'm not the only one who has seen this: 

- https://stackoverflow.com/questions/62491924/developer-log-does-not-print-log-in-the-test 
- https://github.com/Dart-Code/Dart-Code/issues/3653#issuecomment-963065499

It looks like these logs won't print unless the VM service is used?

Reply:
@csells it looks like the way this is working is the best way to do it right now, so I'll remove the TODO.